nixos_systems/staging/wiggle/default.nix

```diff
@@ -40,7 +41,15 @@ in {
 
   programs.nix-ld.enable = true;
 
-  # FIXME: Add networking info here
+  systemd.network = {
+    enable = true;
+    networks = {
+      "10-en" = {
+        matchConfig.name = "en*";
+        networkConfig.DHCP = "yes";
```

Comment:
Won't be DHCP but an internal static IP
```suggestion
        networkConfig.DHCP = "no";
```

Reply:
This'll need a little more here, let me know what the config is in bus when it exists